src/transactions/transactionBuilder/remoteAbi.ts

```diff
@@ -150,7 +150,7 @@ function parseArg(
   // TODO: support uint8array?
   if (param.isAddress()) {
     if (isString(arg)) {
-      return AccountAddress.fromString(arg);
+      return AccountAddress.fromStringRelaxed(arg);
```

Comment:
why?

Reply:
Basically, if you pull something out of the API elsewhere and we don't type it for you, it could be missing the 0's.  If you're pulling it out of one system into the other, you want it to be more permissive.

In this case, that occurred a couple times to me when accounts were generated.